src/Native/System.Native/pal_networking.cpp

```diff
@@ -21,16 +21,14 @@ static_assert(PAL_TRY_AGAIN == TRY_AGAIN, "");
 static_assert(PAL_NO_RECOVERY == NO_RECOVERY, "");
 static_assert(PAL_NO_DATA == NO_DATA, "");
 static_assert(PAL_NO_ADDRESS == NO_ADDRESS, "");
-static_assert(sizeof(uint8_t) == sizeof(char), ""); // We make casts from uint8_t to char for OS functions, make sure it's legal
+static_assert(sizeof(uint8_t) == sizeof(char),
+              ""); // We make casts from uint8_t to char for OS functions, make sure it's legal
```

Comment:
I believe this was moved due to the line-length limitation where the comment pushed the line over the character limit so it moved the argument and the comment to a new line